### PR TITLE
Update hyb2onc2isis to support newer image format

### PR DESCRIPTION
**Pull request by krlberry**
##  Description
Updates `hyb2onc2isis` to support more a recent format of Hayabusa2 fits images with different label keywords needed by ISIS. Most of this PR is just adding the changes to translation files identified by @foobarbecue (Thanks!) in #3698 to ISIS.

To get the resulting cubes spiceinit-ing, it was also necessary to make several updates to the hayabusa2 kernel area.

## Related Issue
#3698

## Motivation and Context
Updates `hyb2onc2isis` to work with more recently released Hayabusa2 fits files.The label format has changed enough that the translation files needed to be updated.

## How Has This Been Tested?
Old hayabusa2 tests pass and added a test for the new format.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by krlberry** [comment]
The jenkins failure: 1200 -` hyb2onc2isis_app_test_newFits (Failed)` for this PR is expected because I didn't check in the new test data yet.

File: isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+commands:
+	$(APPNAME) from=$(INPUT)/hyb2_onc_20180710_060508_tvf_l2a.fit  \
+	           to=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub   \
+	           >& /dev/null;
+	catlab     from=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub \
+	           to=$(OUTPUT)/labels.pvl                             \
+	           >& /dev/null;
+	catoriglab from=$(OUTPUT)/hyb2_onc_20180710_060508_tvf_l2a.cub \
+	           to=$(OUTPUT)/origLab.pvl                            \
+	           >& /dev/null;
```
Comment (jessemapel) [inline isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile:18]: Is the test data for this already checked in?

Comment (krlberry) [reply isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile:18]: No.

**Comment by jessemapel** [comment]
[hyb2onc2isis needs updating to deal with new Hyabusa2 FITS image labels](https://trello.com/c/8Zq4TPTu/80-hyb2onc2isis-needs-updating-to-deal-with-new-hyabusa2-fits-image-labels)

File: isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile (hunk 1 above)
Comment (jessemapel) [reply isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile:18]: For new tests, it's okay to check in the test data early as it won't conflict with other PRs or dev.

Comment (krlberry) [reply isis/src/hayabusa2/apps/hyb2onc2isis/tsts/newFits/Makefile:18]: Forgot about that, good point.

**Review by jessemapel — APPROVED** [review]
This looks good. I'm still going to watch for CI to finish though

**Pull request merged**